Docs landing (#2298)
slint-ui/slint: README.md

### Accessibility

Slint supports keyboard based navigation of many widgets, and user interfaces
are scalable. The basic infrastructure for assistive technology like screen
readers is in place, but currently requires the Qt backend.

> We are aware that more work is needed to get best-of-class support for users

Reviewer: ```suggestion
We're aware that more work is needed to get best-of-class support for users
```

## Documentation

> Check out the [Slint Language Documentation](https://slint-ui.com/docs/slint)
> for more details.

Reviewer: ```suggestion
For more details, check out the [Slint Language Documentation](https://slint-ui.com/docs/slint).
```
